app/index.html

```diff
@@ -12,52 +12,60 @@
   <script src="https://d3js.org/d3.v3.min.js"></script>
   <script src="node_modules/papaparse/papaparse.min.js"></script>
   <!--Sources for svg/png download -->  
+  <script src="//d3js.org/d3.v4.min.js"></script>
```

Comment:
That broke the rendering last time, it was also found in a previous pull request.